Week1/Day2/ExercisesXP/code.py

```diff
@@ -0,0 +1,119 @@
+#Exercise1
+my_fav_numbers = {7,88,24,23,10}
+my_fav_numbers.add(33)
+my_fav_numbers.add(16)
+print(my_fav_numbers)
+my_fav_numbers.remove(10) #we can delete last number only if we know what is last number
+print(my_fav_numbers)
+friend_fav_numbers={21,9,3,15}
+our_fav_numbers = friend_fav_numbers.union(my_fav_numbers)
+print(our_fav_numbers)
+print("---------")
+
+#Exercise2
+sample_tuple = (1,2,3,4,5,6,7,8,9) #it is not possible to add more integers to a tuple
+
+#Exercise3
+basket = ["Banana", "Apples", "Oranges", "Blueberries"]
+basket.remove("Banana")
+basket.remove("Blueberries")
+basket.append("Kiwi")
+basket.insert(0, "Apples")
```

Comment:
any other way to add an element at the start of the list?

Reply:
basket = ["Apples", *basket]